ubcpi/static/html/ubcpi.html

```diff
@@ -22,11 +22,11 @@ <h3 id="pi-question-h" class="question-text" style="display:inline;">{{display_n
             <div role="group" aria-label="Progress Indicator">
                 <span id="reflect" ng-if="rc.status() == rc.ALL_STATUS.ANSWERED">
                     <ol style="background:#f8f8f8;border:2px solid #ddd;margin:1em 0;padding-top:.3em;">
-                        <li style="display:inline-block;width:30%;text-align:center;color:#414141;padding-top:.3em"><span class="fa fa-check-circle" aria-hidden="true" style="font-size:1.4em;"></span><br><span translate>Answer</span><span class="sr" translate>, Completed</span></li>
-                            <span class="fa fa-angle-double-right" style="color:#ddd;vertical-align:15%;font-size:2.2em" aria-hidden="true"></span>
-                        <li style="display:inline-block;width:30%;text-align:center;font-weight:bold;color:#111111;padding-top:.3em;"><span class="fa fa-arrow-circle-down" aria-hidden="true" style="font-size:1.4em;" ></span><br><span translate>Reflection</span><span class="sr" translate>, In Progress</span></li>
-                            <span class="fa fa-angle-double-right" style="color:#ddd;vertical-align:15%;font-size:2.2em" aria-hidden="true"></span>
-                        <li style="display:inline-block;width:30%;text-align:center;color:#414141"><span class="fa fa-circle-o" aria-hidden="true" style="font-size:1.4em;"></span><br><span translate>Results</span><span class="sr" translate>, Upcoming</span></li>
+                        <li style="display:inline-block;width:30%;text-align:center;color:#414141;padding-top:.3em"><span class="fa fa-check-circle" aria-hidden="true" style="font-size:1.4em;"></span></span><br>Answer<span class="sr">, Completed</span></li>
```

Comment:
Sorry; I had originally made my changes on a branch that did not branch off of your most recent changes, and when I was incorporating everything, I didn't realize that your changes were touching the same parts of the code as mine.

Please see the newest change; I also found and fixed a bug with the display of text for the "Next Step" button.